setup-scripts/health-checker.py

```diff
@@ -43,6 +43,7 @@
 setup()
 
 
+# run_checks runs the health checks
```

Comment:
Let's remove this from the PR once this is ready to be merged

```suggestion
```

Reply:
These were added to prove that the linters are properly triggered.